Cycles_task3/Cycles_task3/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace Cycles_task3
+//Дано натуральное число. Подсчитать кол-во вхождений его максимальной цифры
+{
+    class Maxima {
+        int numb;
+        int max = 0;
+        int cmax = 0;
+        public void GetNumber(int n)
+        {
+            numb = n;
+        }
+
+        public int ShowNumber() {
+            return numb;
+        }
+
+        public int GetMax() {
+            int rest = numb;
+            do {
+                if (max < (rest % 10)) {
+                    max = rest % 10;
+                }
+                rest /= 10;
+            }
+            while (rest > 0);
+            return max;
+        }
+
+        public int CountMax() {
+            int rest = numb;
+            do {
+                if (rest % 10 == max) {
+                    cmax++;
+                }
+                rest /= 10;
+            }
+            while (rest > 0);
+            return cmax;
+        }
+    }
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Maxima m1 = new();
+            m1.GetNumber(1928991637);
```

Comment:
it seems that number can be set up through constructor, no need to create additional method

Reply:
fixed accordingly